projects/pastanaga-angular/src/lib/tabs/tabs-list.component.ts

```diff
@@ -7,32 +7,43 @@ import {
     ChangeDetectorRef,
     ElementRef,
     OnDestroy,
+    Input,
+    Renderer2,
 } from '@angular/core';
 import { TabItemComponent } from './tab-item.component';
-import { BehaviorSubject, fromEvent, interval, Subject } from 'rxjs';
+import { fromEvent, interval, Subject } from 'rxjs';
 import { detectChanges } from '../common';
 import { debounce, takeUntil } from 'rxjs/operators';
+import { coerceBooleanProperty } from '@angular/cdk/coercion';
 
 @Component({
     selector: 'pa-tabs',
     templateUrl: 'tabs-list.component.html',
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class TabsListComponent implements AfterContentInit, OnDestroy {
+    @Input() set fullWidth(full: boolean) {
+        this.fullWidthTabItems = coerceBooleanProperty(full);
+        this.updateSlider();
+    }
```

Comment:
the default is true: full width by default. so the Input should be something allowing to not have full width, so `noFullWidth` for example. I guess in most cases this won't be set dynamically: we want either full width or no full width on an element. Doing it this way will allow us to have code like
```html
<pa-tabs noFullWidth>…
```